controller/workitem.go

```diff
@@ -282,12 +282,12 @@ func (c *WorkitemController) Create(ctx *app.CreateWorkitemContext) error {
 	return application.Transactional(c.db, func(appl application.Application) error {
 		//verify spaceID:
 		// To be removed once we have endpoint like - /api/space/{spaceID}/workitems
-		_, spaceLoadErr := appl.Spaces().Load(ctx, ctx.SpaceID)
-		if spaceLoadErr != nil {
+		exists, err := appl.Spaces().Exists(ctx, ctx.SpaceID.String())
+		if !exists || err != nil {
 			return jsonapi.JSONErrorResponse(ctx, errors.NewBadParameterError("space", "string").Expected("valid space ID"))
```

Comment:
Yeah, that can work meaning to simply have here `jsonapi.JSONErrorResponse(ctx, err)`. Is that what you mean ?